tests/settings_tests/tests.py

```diff
@@ -424,6 +424,18 @@ def test_set_with_xheader_right(self):
         req.META["HTTP_X_FORWARDED_PROTO"] = "https"
         self.assertIs(req.is_secure(), True)
 
+    @override_settings(SECURE_PROXY_SSL_HEADER=("HTTP_X_FORWARDED_PROTO", "https"))
+    def test_set_with_xheader_right_multiple(self):
+        req = HttpRequest()
+        req.META["HTTP_X_FORWARDED_PROTO"] = "https, http"
```

Comment:
Assuming I know my proxy setup, can't I just expect this string, `"https, http"` for `"HTTP_X_FORWARDED_PROTO"`? 🤔

Reply:
If it's anything like XFF, yes, it _could_ have the `", "` _or_ `","` (no space). I'd _assume_ it has many of the same perils and pitfalls as XFF; there was a remarkably in-depth [post about the latter](https://adam-p.ca/blog/2022/03/x-forwarded-for/) the other day which was illuminating and confusing, some of which probably apply to XFP too?